Samples/TdLib.Samples.GetChats/Program.cs

```diff
@@ -3,8 +3,9 @@
 using System.Threading;
 using System.Threading.Tasks;
 using TDLib.Bindings;
+using TdLib;
 
-namespace TdLib.Samples.GetChats;
```

Comment:
Well, technically, the project is called `TDLib.Samples.GetChats` (with capital `D`), and I expect the namespace to be called the same.

But let's wait for #114 before settling on this.